packages/web/components/discover/filter/discover-filter-from.component.tsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import { Form, Select, DatePicker, Slider, Button, Checkbox } from 'antd';
+import { FilterDiscoverySectionComponent } from './discover-filter-section.component';
+import {
+  GetDiscoverQueryVariables,
+  useGetLanguagesQuery,
+  useGetGenresQuery,
+} from '../../../utils/graphql';
+interface DiscoverFilterFormComponentProps {
+  params: GetDiscoverQueryVariables;
+  onFinish: (formParams: GetDiscoverQueryVariables) => void;
+}
+export function DiscoverFilterFormComponent(
+  props: DiscoverFilterFormComponentProps
+) {
+  const languagesQuery = useGetLanguagesQuery();
+  const genresQuery = useGetGenresQuery();
+
+  const TMDBLanguages = languagesQuery.data?.languages;
+  const TMDBMovieGenres = genresQuery.data?.genres.movieGenres;
+
+  const [form] = Form.useForm();
+  const { Option } = Select;
+  const { onFinish } = props;
+
+  const options = TMDBLanguages?.map((l) => (
+    <Option key={l.code} value={l.code}>
+      {l.language}
+    </Option>
+  ));
+
+  const onSearch = (values: GetDiscoverQueryVariables) => {
+    onFinish(values);
+  };
+
+  function formatter(score: number) {
+    return `${score}%`;
+  }
```

Comment:
define this function outside of the render, we do not need to re-define it at every render.